iothub/device/tests/TransportSettingsTests.cs

```diff
@@ -1,271 +1,333 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information
 
+using System;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Azure.Devices.Client.ApiTest;
+using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
 namespace Microsoft.Azure.Devices.Client.Test
 {
-    using System;
-    using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Client.ApiTest;
-    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
-    using Microsoft.VisualStudio.TestTools.UnitTesting;
-
     [TestClass]
     [TestCategory("Unit")]
     public class TransportSettingsTests
     {
-        const string LocalCertFilename = "..\\..\\Microsoft.Azure.Devices.Client.Test\\LocalNoChain.pfx";
-        const string LocalCertPasswordFile = "..\\..\\Microsoft.Azure.Devices.Client.Test\\TestCertsPassword.txt";
+        private const string LocalCertFilename = "..\\..\\Microsoft.Azure.Devices.Client.Test\\LocalNoChain.pfx";
```

Comment:
_camelCase

Reply:
not for const. The editorconfig file has the rules in it, and the IDE will suggest when the name needs to change.